src/styles/main.scss

```diff
@@ -1,3 +1,45 @@
+$size: 300px;
+
+html {
```

Comment:
I think styles for body and HTML we have to separate into different block (normalize)

Reply:
It's ok for this task